ci-bench/src/bench_lib/mod.rs

```diff
@@ -0,0 +1,17 @@
+mod benchmark;
+mod cachegrind;
+mod compare;
+mod runner;
+
+pub use benchmark::{validate_benchmarks, Benchmark, ReportingMode};
+pub use cachegrind::CachegrindRunner;
+pub use compare::compare;
+pub use runner::{run_all, run_single};
+
+pub fn black_box<T>(dummy: T) -> T {
```

Comment:
What's the story with this bit? Could you add a comment motivating this fn? 

I have passing familiarity with `core::hint::black_box` and wonder why something custom is required here.

Reply:
I didn't know `core::hint::black_box` was stabilized last December! I'd gladly use it instead of a custom implementation if we aren't required to support a lower Rust version in the benchmarking code (currently being discussed [here](https://github.com/rustls/rustls/pull/1404#discussion_r1297432171)).

Reply:
Update: I've updated the code to use the `black_box` from the standard library

Reply:
Awesome, thanks!